tests/testthat/test-course.R

```diff
@@ -154,7 +154,7 @@ test_that("create_download_url() works", {
 })
 
 test_that("normalize_url() prepends https:// (or not)", {
-  expect_error(normalize_url(1), "is\\.character.*not TRUE")
+  expect_error(normalize_url(1), "must be a valid name")
```

Comment:
Let's go ahead and make this a snapshot test.
```suggestion
  expect_snapshot(normalize_url(1), error = TRUE)
```
Usually, when I touch existing regex-y tests of error messages, I convert them to a snapshot test (vs. updating the regex). Most of them originate from a time before snapshot tests.